app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -296,6 +305,30 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
         return settings().lastKnownMode
     }
 
+    /**
+     * Determines whether the activity should be pushed to be backstack (i.e., 'minimized' to the recents
+     * screen) upon starting.
+     * @param intent - The intent that started this activity. Is checked for having the 'START_IN_RECENTS_SCREEN'-extra.
+     * @return true if the activity should be started and pushed to the recents screen, false .
+     */
+    private fun shouldStartInRecentsScreen(intent: Intent?): Boolean {
```

Comment:
Can you help me understand this function a bit better? From what I can tell this is used to put the app in the app switcher. Maybe a clearer name like `shouldAddToRecentsScreen` or something like that?